src/pyload/webui/app/templates/js/dashboard.js

```diff
@@ -163,7 +163,7 @@ var LinkEntry = new Class({
 
         this.elements.remove.addEvent('click', function(){
             new Request({
-                method: 'get', url: '/json/abort_link/' + this.id
+                method: 'get', url: window.location.pathname + "/../json/abort_link/" + this.id
```

Comment:
Shouldn't this be:
`method: 'get', url: "{{url_for('json.abort_link')}}"  + this.id`?